simulator/ns-3.35/src/point-to-point/model/switch-mmu.cc

```diff
@@ -85,8 +87,8 @@ SwitchMmu::SwitchMmu(void) {
 	for (uint32_t port = 0; port < pCnt; port++) {
 		for (uint32_t q = 0; q < qCnt; q++) {
 			// buffer configuration.
-			reserveIngress[port][q] = 1248; // Per queue reserved buffer at ingress
```

Comment:
Headroom buffer is also reserved. Actually here you mean minimum guaranteed buffer. Can you rename these variables to "minXXX".

Reply:
Can we keep it as a todo for later? This will trigger changes in many places and also function names for SetReserved.

For now, it seems easy to distinguish since there are two names, "reserved" and "headroom".